Makefile

```diff
@@ -36,11 +40,13 @@ PACKAGE_JSON_VERSION = $(shell jq -r '.version' package.json)
 ifeq ($(RELEASE_TYPE),production)
 APPLICATION_VERSION = $(PACKAGE_JSON_VERSION)
 S3_BUCKET = resin-production-downloads
+BINTRAY_COMPONENT = etcher
 endif
 ifeq ($(RELEASE_TYPE),snapshot)
 CURRENT_COMMIT_HASH = $(shell git log -1 --format="%h")
 APPLICATION_VERSION = $(PACKAGE_JSON_VERSION)+$(CURRENT_COMMIT_HASH)
 S3_BUCKET = resin-nightly-downloads
+BINTRAY_COMPONENT = etcher-devel
```

Comment:
...with this then being `BINTRAY_COMPONENT = $(APPLICATION_NAME_LOWERCASE)-devel` ?